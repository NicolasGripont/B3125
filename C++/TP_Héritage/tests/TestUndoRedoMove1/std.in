UNDO
REDO
S seg1 1 1 7 5
R rect1 2 1 11 5
OR or1 seg1 rect1
PC pc1 0 0 4 8 8 -1
OI oi1 or1 pc1
LIST
MOVE oi1 1 2
LIST
MOVE or 2 3
MOVE or1 2 3
LIST
MOVE seg1 -1 1
LIST
MOVE rect1 -2 3
LIST
MOVE pc1 1 -2
LIST
UNDO
LIST
UNDO
LIST
UNDO
LIST
UNDO
LIST
UNDO
LIST
REDO
LIST
REDO
LIST
EXIT