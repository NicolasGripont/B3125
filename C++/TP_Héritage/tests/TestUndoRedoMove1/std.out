OK
OK
OK
OK
OK
OK
OK
OI oi1 2
	OR or1 2
		S seg1 1 1 7 5
		R rect1 2 1 11 5
	PC pc1 0 0 4 8 8 -1
PC pc1 0 0 4 8 8 -1
R rect1 2 1 11 5
OR or1 2
	S seg1 1 1 7 5
	R rect1 2 1 11 5
S seg1 1 1 7 5
OK
OI oi1 2
	OR or1 2
		S seg1 2 3 8 7
		R rect1 3 3 12 7
	PC pc1 1 2 5 10 9 1
PC pc1 0 0 4 8 8 -1
R rect1 2 1 11 5
OR or1 2
	S seg1 1 1 7 5
	R rect1 2 1 11 5
S seg1 1 1 7 5
ERR
OK
OI oi1 2
	OR or1 2
		S seg1 2 3 8 7
		R rect1 3 3 12 7
	PC pc1 1 2 5 10 9 1
PC pc1 0 0 4 8 8 -1
R rect1 2 1 11 5
OR or1 2
	S seg1 3 4 9 8
	R rect1 4 4 13 8
S seg1 1 1 7 5
OK
OI oi1 2
	OR or1 2
		S seg1 2 3 8 7
		R rect1 3 3 12 7
	PC pc1 1 2 5 10 9 1
PC pc1 0 0 4 8 8 -1
R rect1 2 1 11 5
OR or1 2
	S seg1 3 4 9 8
	R rect1 4 4 13 8
S seg1 0 2 6 6
OK
OI oi1 2
	OR or1 2
		S seg1 2 3 8 7
		R rect1 3 3 12 7
	PC pc1 1 2 5 10 9 1
PC pc1 0 0 4 8 8 -1
R rect1 0 4 9 8
OR or1 2
	S seg1 3 4 9 8
	R rect1 4 4 13 8
S seg1 0 2 6 6
OK
OI oi1 2
	OR or1 2
		S seg1 2 3 8 7
		R rect1 3 3 12 7
	PC pc1 1 2 5 10 9 1
PC pc1 1 -2 5 6 9 -3
R rect1 0 4 9 8
OR or1 2
	S seg1 3 4 9 8
	R rect1 4 4 13 8
S seg1 0 2 6 6
OK
OI oi1 2
	OR or1 2
		S seg1 2 3 8 7
		R rect1 3 3 12 7
	PC pc1 1 2 5 10 9 1
PC pc1 0 0 4 8 8 -1
R rect1 0 4 9 8
OR or1 2
	S seg1 3 4 9 8
	R rect1 4 4 13 8
S seg1 0 2 6 6
OK
OI oi1 2
	OR or1 2
		S seg1 2 3 8 7
		R rect1 3 3 12 7
	PC pc1 1 2 5 10 9 1
PC pc1 0 0 4 8 8 -1
R rect1 2 1 11 5
OR or1 2
	S seg1 3 4 9 8
	R rect1 4 4 13 8
S seg1 0 2 6 6
OK
OI oi1 2
	OR or1 2
		S seg1 2 3 8 7
		R rect1 3 3 12 7
	PC pc1 1 2 5 10 9 1
PC pc1 0 0 4 8 8 -1
R rect1 2 1 11 5
OR or1 2
	S seg1 3 4 9 8
	R rect1 4 4 13 8
S seg1 1 1 7 5
OK
OI oi1 2
	OR or1 2
		S seg1 2 3 8 7
		R rect1 3 3 12 7
	PC pc1 1 2 5 10 9 1
PC pc1 0 0 4 8 8 -1
R rect1 2 1 11 5
OR or1 2
	S seg1 1 1 7 5
	R rect1 2 1 11 5
S seg1 1 1 7 5
OK
OI oi1 2
	OR or1 2
		S seg1 1 1 7 5
		R rect1 2 1 11 5
	PC pc1 0 0 4 8 8 -1
PC pc1 0 0 4 8 8 -1
R rect1 2 1 11 5
OR or1 2
	S seg1 1 1 7 5
	R rect1 2 1 11 5
S seg1 1 1 7 5
OK
OI oi1 2
	OR or1 2
		S seg1 2 3 8 7
		R rect1 3 3 12 7
	PC pc1 1 2 5 10 9 1
PC pc1 0 0 4 8 8 -1
R rect1 2 1 11 5
OR or1 2
	S seg1 1 1 7 5
	R rect1 2 1 11 5
S seg1 1 1 7 5
OK
OI oi1 2
	OR or1 2
		S seg1 2 3 8 7
		R rect1 3 3 12 7
	PC pc1 1 2 5 10 9 1
PC pc1 0 0 4 8 8 -1
R rect1 2 1 11 5
OR or1 2
	S seg1 3 4 9 8
	R rect1 4 4 13 8
S seg1 1 1 7 5